src/foam/nanos/u2/navigation/SignUp.js

```diff
@@ -37,7 +37,8 @@ foam.CLASS({
     { name: 'EMAIL_ERR', message: 'Valid email required' },
     { name: 'EMAIL_AVAILABILITY_ERR', message: 'This email is taken. Please try another.' },
     { name: 'USERNAME_EMPTY_ERR', message: 'Username required' },
-    { name: 'USERNAME_AVAILABILITY_ERR', message: 'This username is taken. Please try another.' }
+    { name: 'USERNAME_AVAILABILITY_ERR', message: 'This username is taken. Please try another.' },
+    { name: 'PASSWORD_ERR', message: 'Password should be at least 6 characters.' }
```

Comment:
What was the original error? And why is that not translated already?  Just curious. I don't like the validation value '6' hard coded into the message.

Reply:
the original error comes from (I think) password property itself, which is code in foam.core.StringPropertyValidationRefinement(this error message is constructed inside), which is not being translated.

Reply:
The original error comes from password property itself, which is in 'foam.core.StringPropertyValidationRefinement'(the error message is constructed inside), which I don't think it is being translated.

Reply:
Let's leave the the PASSWORD_ERR out for now. we need a better way to deal with these.